benchmark/python/tpch_functional_queries.py

```diff
@@ -166,6 +184,23 @@ def q8(self):
             .sort(col("o_year")) \
             .show()
 
+    def q8(self):
+        getYear = udf(lambda x: x[0:4], StringType())
+        decrease = udf(lambda x, y: x * (1 - y), FloatType())
+        isBrazil = udf(lambda x, y: (y if (x == "BRAZIL") else 0), FloatType())
+
+        self.q8Common(getYear, decrease, isBrazil)
+        
+
+    def q8a(self):               
+        getYear = udf(lambda x: x[0:4], StringType())
```

Comment:
Looks like `getYear` and `isBrazil` can be moved to common?

Reply:
Done!